### PR TITLE
Remove flask-limiter and redis

### PR DESCRIPTION
**Pull request by micahflee**
I think it may be a good idea to remove Flask-Limiter and Redis from the app until we're confident it solves a problem we actually have.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -10,19 +10,16 @@ readme = "README.md"
 python = "^3.11"
 cryptography = "^42.0.5"
 flask = "^3.0.2"
-flask-limiter = { extras = ["redis"], version = "^3.7.0" }
 flask-migrate = "^4.0.7"
 flask-sqlalchemy = "^3.1.1"
 flask-wtf = "^1.2.1"
-gunicorn = "^22.0.0"
```
Comment (jeremywmoore) [inline pyproject.toml:17]: I don't think we want to remove gunicorn

**Review by jeremywmoore — COMMENTED** [review]
Looks good except for removing the gunicorn dependency, was that intentional?

File: pyproject.toml (hunk 1 above)
Comment (micahflee) [reply pyproject.toml:17]: yeah, that was not intentional

**Comment by micahflee** [comment]
After merging #368, I'm merged main into this branch, and also added back gunicorn.

File: pyproject.toml (hunk 1 above)
Comment (jeremywmoore) [reply pyproject.toml:17]: I see the merge commit but diff still shows this as removed

**Review by jeremywmoore — APPROVED** [review]

**Pull request merged**